OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -234,7 +234,7 @@ private void trackAATestCourseOutline() {
             final FirebaseRemoteConfig firebaseRemoteConfig = FirebaseRemoteConfig.getInstance();
             firebaseRemoteConfig.fetchAndActivate().addOnCompleteListener(task -> {
                 final String group = firebaseRemoteConfig.getString(Analytics.Keys.AA_EXPERIMENT);
-                if (!TextUtils.isEmpty(group)) {
+                if (!TextUtils.isEmpty(group) && environment.getLoginPrefs().getCurrentUserProfile() != null) {
```

Comment:
`environment.getLoginPrefs().getCurrentUserProfile()` better to store in local variable and then use it.

Reply:
Done